packages/loader/container-loader/src/loader.ts

```diff
@@ -396,6 +398,8 @@ export class Loader extends EventEmitter implements ILoader {
                     resolvedAsFluid);
         }
 
+        this.emit("containerLoaded", container, newContainer);
```

Comment:
>emit [](start = 13, length = 4)

i don't think we should emit when getting some cache, only when newly created

Reply:
i think we may just want to not do this change in this PR. there is some good discussion in #5114, and adding this the wrong way could make it harder to trim/remove the loader from the container runtime